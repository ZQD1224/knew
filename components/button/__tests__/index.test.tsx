import React from 'react';
import { mount } from 'enzyme';
import Button from '..';
import mountTest from '../../../tests/shared/mountTest';

describe('Button', () => {
  mountTest(Button);

  it('renders correctly', () => {
    expect(<Button>Follow</Button>).toMatchRenderedSnapshot();
  });

  it('mount correctly', () => {
    expect(() => mount(<Button>Follow</Button>)).not.toThrow();
  });
});
