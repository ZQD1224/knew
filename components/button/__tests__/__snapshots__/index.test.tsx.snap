// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  class="ant-btn ant-btn-primary"
  type="button"
>
  <span>
    按钮Follow
  </span>
</button>
`;
